lib/components/settings/update_manager.dart

```diff
@@ -190,13 +215,36 @@ abstract class UpdateManager {
 
     OpenFilex.open(file.path);
   }
+
+  static Future<String> getNewestVersionChangelog() async {
+    final int newestVersion;
+    newestVersion = await getNewestVersion() ?? 0;
+    return getChangelogText(newestVersion);
+  }
+
+  static Future<String> getChangelogText(int newestVersion) async {
```

Comment:
Are `getChangelogText` and `downloadChangelogText` separate functions intentionally?